src/screens/Reports/Report.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { FinancialSummaryRecord } from '../../lib/airtable/interface';
+import BaseScreen from '../../components/BaseComponents/BaseScreen';
+import BaseScrollView from '../../components/BaseComponents/BaseScrollView';
+import OutlinedCardList from '../../components/OutlinedCardList';
+import { Typography, List, ListItem, ListItemText } from '@material-ui/core';
+import { createStyles, Theme, withStyles } from '@material-ui/core/styles';
+import { RouteComponentProps } from 'react-router-dom';
+import { selectCurrentSiteInformation, round } from '../../lib/redux/siteData';
+import { Person as PersonIcon, ShoppingCart as ShoppingCartIcon, Build as BuildIcon, Warning as WarningIcon } from '@material-ui/icons';
+import ListItemWrapper from '../../components/ListItemWrapper';
+import { useSelector } from 'react-redux';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    section: {
+      padding: '10px 5px',
+    },
+    header: {
+      display: 'inline-flex',
+      alignItems: 'center',
+    },
+    list: {
+      borderLeft: `5px solid ${theme.palette.primary.main}`,
+      padding: '2px 8px',
+      margin: '4px',
+    },
+    icon: {
+      marginRight: '5px',
+    },
+  });
+
+interface ReportProps extends RouteComponentProps {
+  classes: { section: string; header: string; list: string; icon: string; };
+  location: any;
+}
+
+interface Info {
+  label: string;
+  value: string;
+}
+
+function Report(props: ReportProps) {
+  const { classes, match } = props;
+  const report = props.location.state.report;
+  const deadline = props.location.state.deadline;
+  const currentSite = useSelector(selectCurrentSiteInformation);
+
+  // Customers
+  const collected = `${round(report.totalAmountCollected)} Ks`;
+  const billed = `${round(report.totalAmountBilled)} Ks`;
+  const outstanding = `${round(report.totalAmountBilled - report.totalAmountCollected)} Ks`;
+  const paymentRate = report.totalCustomersBilled ? `${round(report.totalCustomersPaid / report.totalCustomersBilled * 100)}%` : '--';
+
+  // Inventory
+  const approved = `${report.inventoryAmountApproved.toString()} Ks`;
+  const rejected = `${report.inventoryAmountDenied.toString()} Ks`;
+
+  // TODO: Add calculations for maintenance & incidents
+  const info = {
+    profit: [{ number: report.totalProfit, label: 'Profit', unit: 'Ks' }],
+    customers: [
+      { label: 'Collected', value: collected},
+      { label: 'Billed', value: billed},
+      { label: 'Outstanding', value: outstanding },
+      { label: 'Payment Rate', value: paymentRate },
+    ],
+    inventory: [
+      { label: 'Approved', value: approved },
+      { label: 'Rejected', value: rejected },
+    ],
+    maintenance: [
+      { label: 'Completed', value: '0' },
+      { label: 'Incomplete', value: '0' },
+    ],
+    incidents: [
+      { label: 'Solved', value: '0' },
+      { label: 'Unsolved', value: '0' },
+    ],
+  }
```

Comment:
It'd be easier to read if we didn't have this wrapping `info` object. We could just define the variables `profitInfo`, `customersInfo`, `inventoryInfo`, `maintenanceInfo`, and `incidentsInfo`, etc and pass them to `getContentsList` to render. 

I think we should delete the `info` object because the variable name is not very telling to the reader and it's hard to interpret at first glance because it's such a large object.